src/test/shell/bazel/platforms_test.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
nit: last time I checked, it already was 2019

Reply:
:)